README.md

```diff
@@ -1,97 +1,142 @@
-###Journalduhacker Project
+# Gambe.ro
 
-This is the source code of the website operating at
-[https://www.journalduhacker.net](https://www.journalduhacker.net).  It is a Rails 4 codebase and uses a
-SQL (MariaDB in production) backend for the database and Sphinx for the search
-engine.
+[gambe.ro](https://www.gambe.ro) is an italian lobste.rs clone. We aim to build a strong community of italian programmers where constructive discussions can take place.
```

Comment:
clone mi sembra una parola inadatta. Piuttosto qualcosa tipo "is an italian community built using lobste.rs software".